keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/constraints/CanaryConstraintConfigurationProperties.kt

```diff
@@ -0,0 +1,9 @@
+package com.netflix.spinnaker.keel.constraints
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties("defaults.canary")
```

Comment:
Any opinions on this property scope? I was going back and forth between `defaults.canary` and `defaults.constraint.canary`

Reply:
I vote shorter.

Reply:
Actually, given the name of the properties class, I changed my mind.